internal/jet/literal_expression.go

```diff
@@ -326,6 +326,102 @@ func formatNanoseconds(nanoseconds ...time.Duration) string {
 	return ""
 }
 
+// ---------------------------------------------------//
+type rangeLiteral struct {
+	rangeInterfaceImpl
+	literalExpressionImpl
+}
+
+func RawRange(raw string, namedArgs ...map[string]interface{}) RangeExpression {
+	return RangeExp(Raw(raw, namedArgs...))
+}
+
+func NumRange(lowNum, highNum *int, bounds string) RangeExpression {
+	validateBounds(bounds)
+
+	low := ""
+	if lowNum != nil {
+		low = fmt.Sprint(*lowNum)
+	}
+	high := ""
+	if highNum != nil {
+		high = fmt.Sprint(*highNum)
+	}
+	return RawRange(fmt.Sprintf("%clowerNum,higherNum%c", bounds[0], bounds[1]), map[string]interface{}{
```

Comment:
`NumRange` should be a [constructor function]( https://www.postgresql.org/docs/current/rangetypes.html#RANGETYPES-CONSTRUCT), not a literal.
`numrange(1.0, 14.0, '(]');` vs `'[3,7)'::int4range`.

You can use function helper function:

```golang
return RangeExp(NewFunc("numrange", allparameters, nil))
```